docs/devguide/xtypes.rst

```diff
@@ -922,38 +998,38 @@ Reading Collections of Basic Types
 
 Besides a list of methods for getting values of members of basic types, DynamicData also defines methods for reading sequence members.
 In particular, for each method that reads value from a basic type, there is a counterpart that reads a sequence of the same basic type.
-For instance, get_int32_value reads the value from a member of type long, and get_int32_values reads the value from a member of type sequence<long>.
-For the member ul_seq_field in our example, its value can be read as follows.
+For instance, ``get_int32_value`` reads the value from a member of type ``int32``/``long``, and ``get_int32_values`` reads the value from a member of type ``sequence<int32>``.
+For the member ``ul_seq_field`` in our example, its value can be read as follows.
 
-::
+.. code-block:: cpp
 
-    CORBA::ULongSeq my_ul_seq;
+    DDS::UInt32Seq my_ul_seq;
```

Comment:
Has anyone tried this in a test?  I'm not sure it will directly work because different sequence typedefs in IDL generate distinct mapped classes.

Reply:
The `get_*_values` are defined using `DDS::*Seq`.

https://github.com/OpenDDS/OpenDDS/blob/3734398337a043e067fe6439e09100f974962e94/dds/DdsDynamicData.idl#L307-L308

Reply:
Right, I confused this with the other use of `DDS::` module